README.md

```diff
@@ -1,25 +1,65 @@
 # presentation-microservices-kotlin
-Presentation code and material for micro-services with kotlin
+
+Build microservices wth Kotlin + Spring Boot
+Presented at GDG Kotlin/Everywhere Hong Kong : A Day of Kotlin, 31st Aug 2019
+Slides can be found here
 
 [![master](https://img.shields.io/badge/travis-master-blue.svg)][travis-url][![TravisCI][travis-master-badge]][travis-url]
 
+## Introduction
+
+Two backend services are created in this project, `auth` and `app` respectively. A microservice-based architecture can be achieved locally with docker-compose or Docker. We have also deployed them on Oursky's k8s cluster.
+
 ## Get Started
+
 ##### Generate secret
+
 ```
 cd backend
 make secret
 ```
-##### Start servers
+
+##### Start servers with docker-compose
+
 ```
 cd backend
 docker-compose -p demo up
 ```
+
+##### With Docker
+
+```
+cd backend
+docker build -f Dockerfile.auth . -t demo-auth
+docker run --rm -it -p 8080:8080 -t demo-auth
+# OR
+docker run --rm -it -p 8080:8080 $(docker build -f Dockerfile.auth -q .)
+```
+
+#### Or with on our k8s cluster
+
+Simply curl the APIs by changing the endpoint to `https://present-kotlin.pandawork.com`. Here's an example calling the Sign Up API:
+
+```
+curl -X POST https://present-kotlin.pandawork.com/auth/signup -H "Content-Type: application/json" -d '{"user": "test_user", "pass": "Admin123"}'
+```
+
+#### To see how APIs can be easily created
```

Comment:
Don't direct reader to individual code, if you want to point it out it should be on the slides.